README.md

```diff
@@ -40,7 +40,15 @@ TODO
 
 Use with Up board
 -----------------
-TODO
+On the Up board, there is only a few points you need to check if you have issue with permissions (symptom is usually an error `Could not set necessary configuration of serial port` from the SBUS bridge)
+
+1.  You are using an "upboard" kernel. Check for that using `uname -a`
+      -  If not, go this page and install such a kernel: [https://wiki.up-community.org/Ubuntu](https://wiki.up-community.org/Ubuntu)
+   
+2.  You have set the permissions and installed the add-ons as described here: [https://wiki.up-community.org/Ubuntu#Enable_the_HAT_functionality_from_userspace](https://wiki.up-community.org/Ubuntu#Enable_the_HAT_functionality_from_userspace)
+
+3.  You are using the wrong port name which is usually `/dev/ttyS1` or `/dev/ttyS4`.
```

Comment:
This could be confusing whether the example ports are the wrong or right ones

Reply:
done